CONTRIBUTING.md

```diff
@@ -57,11 +57,6 @@ end with `.stories.tsx`.
 
 **`yarn release`**: publish changed packages.
 
-### Documentation
-
-The documentation site is built with Next.js. If you'd like to contribute to the
```

Comment:
I think it would be a good idea to add link to `chakra-ui-docs` repository under documentation section so that the contributors who are interested in contributing towards documentation can be helped.

Reply:
There already exists a link for the documentation site. 😃
https://github.com/chakra-ui/chakra-ui/blob/b983e4ceb7c1e794c2b901ce16f12016836d40d6/CONTRIBUTING.md?plain=1#L157-L162